user-guide/Cloud_Platform/Catalog/Tutorials/Tutorial_Register_Catalog_Version.md

```diff
@@ -0,0 +1,65 @@
+---
+uid: Tutorial_Register_Catalog_Version
+---
+
+# Registering a new version of a connector in the Catalog
+
+This tutorial demonstrates how to add a new version to a catalog item using the [Catalog API](xref:Register_Catalog_Item).  
+We will be registering our own version of the following example [connector](https://github.com/SkylineCommunications/SLC-C-Example_Rates-Custom), so go ahead and download the solution.
+
+## Prerequisites
+
+- Organization key with permission "Register catalog items".
```

Comment:
![image](https://github.com/user-attachments/assets/bc6befff-3803-4f06-bdf9-bb612420b96e)

Don't see the key as a prerequisite anymore

Reply:
Yeah, same as comment above I guess. You can leave it out if it's documented how to get it, only reason I'd keep a prerequisite is that a user can immediately see if they can follow this tutorial. If they are not an Owner or do not have a Key from an owner, then this tutorial becomes useless to them. Adding it as a prerequisite lets them quickly filter on that. I guess it's a choice to make, rather than an issue to address.